.github/workflows/generate-manual.yaml

```
@@ -0,0 +1,80 @@
+        run: |
+          wget https://gethstore.blob.core.windows.net/builds/geth-alltools-linux-amd64-1.11.5-a38f4108.tar.gz
+          tar -zxvf geth-alltools-linux-amd64-1.11.5-a38f4108.tar.gz
+          sudo mv geth-alltools-linux-amd64-1.11.5-a38f4108/abigen /usr/local/bin/
+
+      - name: Generate Go packages and types
+        run: |
+          yarn generate
+
+      - name: Check for changes
+        id: check_changes
+        run: |
```

actionlint

shellcheck reported issue in this script: SC1046:error:1:1: Couldn't find 'fi' for this 'if'

(shellcheck)